Update 2025-06-17.md (#14)
w3c-cg/webdocs: meetings/2025-06-17.md

# W3C Docs CG call, 2025-06-17

> Present: Lola, Eric M, Florian S, Edward Loveall, Schalk, Estelle, François, Dipika, Will

Author: ```suggestion
Present: Lola Odelola, Eric Meyer, Florian Scholz, Edward Loveall, Schalk Neethling, Estelle Weyl, François Daoust, Dipika Bhattacharya, Will Bamberg
```

4. [Using IRC to generate Minutes](https://github.com/w3c-cg/webdocs/issues/13)

## Notes

Introductions.

## TPAC

- Lola: Friday is the deadline for proposals. Regarding https://github.com/w3c-cg/webdocs/issues/12 - should we have drop-in sessions for WGs, or an MDN contribution workshop?
- Estelle: both!

> - Explainers: help people understand how to write good epxlainers

Reviewer: ```suggestion
- Explainers: help people understand how to write good explainers
```

> - Estelle: should have: (1) CG session for this grouo, face to face and (2) external session. Figure out  how to get tech writers and spec authors to get together (technical networking). For explainers, if this is the rioght group then yes, but it should go soemnwhere

Reviewer: ```suggestion
- Estelle: should have: (1) CG session for this group, face to face and (2) external session. Figure out how to get tech writers and spec authors to get together (technical networking). For explainers, if this is the right group then yes, but it should go somewhere
```

- Lola: TAG owns explainer process.
- Estelle: can Lola ask TAG if they are going to have an explainer session? Otherwise we will.
- Lola: probably they won't because we know that explainers aren't going anywhere.
- Lola might not be at TPAC, if she is, TAG and WebDX will take up much of her time.
- Will: workshop with Web Components people (and anyone else who wants to work with us on their docs)
- Florian: good to have some concrete topics, e.g. meeting with specific WGs like Web Components. Also work on some strategic stuff for this CG. Talk about "docs as conversations" (LLMs) at TPAC and what this means for docs.

> - FRançois: haven't heard back from Web Components about whether they want to meet, usually they want to meet in breakouts. Trying to encourage them to meet in part of the main schedule. Instead of booking slots, andother approach would be to get on WG agenda and go to their meeting. Might get more WG people then. Risk of hosting our own sessions is that only docs CG people would attend.

Reviewer: ```suggestion
- François: haven't heard back from Web Components about whether they want to meet, usually they want to meet in breakouts. Trying to encourage them to meet in part of the main schedule. Instead of booking slots, and other approach would be to get on WG agenda and go to their meeting. Might get more WG people then. Risk of hosting our own sessions is that only docs CG people would attend.
```

- Lola: overall: we want to meet at TPAC, we want a session that's just us, and also to meet with other grouos such as Web Components. At the moment seems like just one session for us, session is 1.5 hours.
- Florian: 2 sessions:
  - One for kicking docs-cg off and discuss how it's been going.
  - One on LLMs. Invite experts to that: Dominique and also perhaps people from copilot.
 So at least 2 sessions, but don't schedule too much so we are free to attend other groups' sessions.

## AI agents

- Lola: At 5 June WebDX meeting: AI agents are being used and it benefits the web for people using them to have correct (up to date) data. Also, we should have docs on using AI agents, and the risks of using them to generate code.
- Florian: if you have docs talking about things you **shouldn't** do, agents don't understand this so well. Lola: this is guidance for humans.

> - FLoriab: also there are new attack vectors, prompt injections.

Reviewer: ```suggestion
- Florian: also there are new attack vectors, prompt injections.
```

> - Schalk: accessiblity is a big challenge for huge codebases that people don't understand. Could have cursor rule that people can use and share. MDN web docs MCP server, your tool can use to get new info. Schalk is experimenting with making a repo of code examples with semantic/accessible HTML, and see if you can train a model on good examples. But it's a big project to make it scale. AI loves aria, struggles with complex things. but ultimately have to improve the underlying model.

Reviewer: ```suggestion
- Schalk: accessibility is a big challenge for huge codebases that people don't understand. Could have Cursor rule that people can use and share. MDN web docs MCP server, your tool can use to get new info. Schalk is experimenting with making a repo of code examples with semantic/accessible HTML, and see if you can train a model on good examples. But it's a big project to make it scale. AI loves aria, struggles with complex things. but ultimately have to improve the underlying model.
```
